README.md

```diff
@@ -1,2 +1,35 @@
-# cloud-foundry-nozzle
-Wavefront Nozzle for Pivotal Cloud Foundry
+# Summary
+Wavefront Nozzle for Pivotal Cloud Foundry (PCF). The nozzle is deployed as an app (cf push) in PCF.
+<br />
+The nozzle takes the data from Loggregator Firehose and forwards those metrics to Wavefront proxy.
+
+# Prerequisites
+* Make sure you have a working Wavefront proxy with accessible <hostname:port> talking to a Wavefront cloud.
```

Comment:
Would we not have this prerequisites once we have the tile also deploy the proxy as an app?